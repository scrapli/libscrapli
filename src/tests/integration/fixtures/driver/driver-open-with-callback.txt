Warning: Permanently added '[localhost]:22022' (ED25519) to the list of known hosts.
(admin@localhost) Password: 
Last login: Fri Aug 29 18:54:34 2025 from 140.82.116.3
eos1>
eos1>
eos1>enable
Password: 
eos1#term len 0
Pagination disabled.
eos1#term width 32767
Width set to 32767 columns.
eos1#