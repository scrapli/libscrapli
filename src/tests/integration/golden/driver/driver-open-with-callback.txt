Warning: Permanently added '[localhost]:22022' (ED25519) to the list of known hosts.
(admin@localhost) Password: 
Last login: Sat Dec 28 20:07:43 2024 from 172.20.20.1
eos1>
eos1>
eos1#term len 0
Pagination disabled.
eos1#
eos1#term width 32767
Width set to 32767 columns.
eos1#